ext/settings.html

```diff
@@ -1960,7 +1960,7 @@ <h1>Yomitan Settings</h1>
             <div class="settings-item-inner">
                 <div class="settings-item-left">
                     <div class="settings-item-label">
-                        Show card tags
+                        Show card tags and flags
```

Comment:
Would it be annoying to rename the setting to `anki.displayTagsAndFlags`?

Reply:
Wouldnt be too bad but idk if it's worth bothering.

Reply:
Changed the option.